src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -205,10 +205,13 @@ public void apply(final VariantContext vc, final ReadsContext readsContext, fina
             // if the current record is adjacent to the previous record and "overlap" them if they are so our set is as
             // small as possible while still containing the same bases.
             final int start = (previousInterval != null && previousInterval.overlapsWithMargin(refInterval, 1)) ?
-                    previousInterval.getEnd() : refInterval.getStart();
-            genomeLocSortedSet.add(genomeLocSortedSet.getGenomeLocParser().createGenomeLoc(refInterval.getContig(), start, vc.getEnd()), true);
+                    Math.min(previousInterval.getStart(), refInterval.getStart()) : refInterval.getStart();
```

Comment:
will this:
```
Math.min(previousInterval.getStart(), refInterval.getStart()) 
```
not always equal `previousInterval.getStart()` given the order of iteration?

Reply:
👍